Vagrantfile

```diff
@@ -67,7 +68,16 @@ else
     # cleanup openstack-kilo repo if required
     yum remove docker-engine -y || :
     yum-config-manager --disable openstack-kilo
-    curl https://get.docker.com | sed s/docker-engine/docker-engine-#{docker_version}/ | bash
+    if [[ #{docker_version} == *"rc"* ]]; then
```

Comment:
These changes will become redundant soon. The Docker engine versioning has changed. The `-cs` edition won't exist any more. The regular Docker engine version will become `-ce` (community edition).

Can we skip these changes for now, please?